libcaf_net/src/tcp_accept_socket.cpp

```diff
@@ -55,6 +55,9 @@ template <int Family>
 expected<tcp_accept_socket> new_tcp_acceptor_impl(uint16_t port,
                                                   const char* addr,
                                                   bool reuse_addr, bool any) {
+  auto bts = [](bool x) { return x ? "true" : "false"; };
+  printf("%s(%d, %s, %s, %s)\n", __func__, (int) port, addr, bts(reuse_addr),
+         bts(any));
```

Comment:
Why is this `printf` here?
```suggestion
```

Reply:
Oops, looks like some leftover debug code. Glad you've found it! :)